src/dataExplorer/components/ScriptQueryBuilder.tsx

```diff
@@ -196,7 +196,7 @@ const ScriptQueryBuilder: FC = () => {
   )
 
   const tsmNewScriptDropDown =
-    isFlagEnabled('influxqlUI') && hasDBRPs() ? (
+    isFlagEnabled('influxqlUI') && hasDBRPs() && CLOUD ? (
```

Comment:
Thanks to the e2e test, I realized InfluxQL should be only available in cloud, and not in OSS